### PR TITLE
Update Lang.php

### PR DESCRIPTION
**Pull request by nouakun**
Hello, 
I'm working on the **Arabic language pack of ExpressionEngine** and the Arabic language is use RTL (Right to Left) script, so I made this function to help to get the language direction.

Example Usage:
`system/ee/ExpressionEngine/View/_shared/header.php`
```<html lang="<?=ee()->lang->code()?>" dir="<?= ee()->lang->direction()?>">```

```
        <?php if (ee()->lang->direction() == 'ltr'):?>
            <?=ee()->view->head_link('css/common.min.css'); ?>
        <?php else:?>
            <?=ee()->view->head_link('css/common-rtl.min.css'); ?>
        <?php endif;?>
```

screencapture-localhost-ee-admin-php-2021-02-07-23_54_50

## Overview

Replace this paragraph with a description of your changes and reasoning. Provide links to external references/discussions if appropriate.

Resolves [#NN](https://github.com/ExpressionEngine/ExpressionEngine/issues/NN).

## Nature of This Change

- [ ] 🐛 Fixes a bug
- [x] 🚀 Implements a new feature
- [ ] 🛁 Refactors existing code
- [ ] 💅 Fixes coding style
- [ ] ✅ Adds tests
- [ ] 👽 Adds new dependency
- [ ] 🔥 Removes unused files / code
- [ ] 🔒 Improves security 

## Is this backwards compatible?

- [x] Yes
- [ ] No

## Documentation

User Guide Pull Request: https://github.com/ExpressionEngine/ExpressionEngine-User-Guide/pulls/NNN

### CODE REVIEW

File: system/ee/legacy/core/Lang.php [hunk 1]
```diff
@@ -470,6 +470,34 @@ public function code($language_name = null)
...
     }
+    
+    
+        /**
+     * Get language direction
+     *
+     * @param null $language_dir
+     * @return string Direction for given language code.
+     */
+    public function direction($language_dir = null)
```
Comment (intoeetive) [inline system/ee/legacy/core/Lang.php]: Should this be instead `public function direction($language_name = null)` ?

**Comment by obfuscode** [comment]
@twinkble We have implemented a new Contributor License Agreement system since you opened your PR. If you could sign it, we can get this PR merged into the core! :)

**Comment by nouakun** [comment]
> @twinkble We have implemented a new Contributor License Agreement system since you opened your PR. If you could sign it, we can get this PR merged into the core! :)

Hello @obfuscode ,
This link not working anymore: https://cla-assistant.io/ExpressionEngine/ExpressionEngine?pullRequest=862
where can I sign it?

**Comment by obfuscode** [comment]
@twinkble That's odd, that link works for me? Maybe it was a temporary glitch?

**Comment by nouakun** [comment]
> @twinkble That's odd, that link works for me? Maybe it was a temporary glitch?

It's working now, and I signed it.

Thanks.

**Comment by intoeetive** [comment]
closing in favor of https://github.com/ExpressionEngine/ExpressionEngine/pull/1929

**Pull request closed without merging**